docs/content/en/cookbook-core.md

```diff
@@ -0,0 +1,363 @@
+  <a href="https://github.com/ergonode/vuems-demo" target="_blank">repository</a>.
+
+</alert>
+
+```vue [@Core/components/Logo.vue]
+<template>
+  <svg> ... </svg>
+</template>
+
+<script>
+export default {
+  name: 'Logo',
+}
+</script>
+
+<style>
+.VuemsLogo {
+  animation: 1s appear;
+  margin: auto;
+  width: 300px;
+}
+@keyframes appear {
+  0% {
+    opacity: 0;
+  }
+}
+</style>
+```
+
+### `Wrapper.vue`
+Create a `Wrapper.vue` file in `component` directory.
+The component will wrap other components.
+
+```vue [@Core/components/Wrapper.vue]
+<template>
+  <div class="wrapper">
+    <Logo />
+    <slot />
+  </div>
+</template>
+
+<script>
+import Logo from '@Core/components/Logo'
+
+export default {
+  name: 'Wrapper',
+  components: {
+    Logo,
+  },
+}
+</script>
+
+<style>
+.wrapper {
+  margin: 0 auto;
+  display: grid;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+}
+</style>
+```
+
+### `Navigation.vue`
+Create a `Navigation.vue` file in `component` directory.
+The component will be used in the layout component.
+
+```vue [@Core/components/Navigation.vue]
+<template>
+  <nav class="navigation">
+    <ul>
+      <li><NuxtLink class="button--green" to="/">Home page</NuxtLink></li>
+      <li><NuxtLink class="button--grey" to="/about">About</NuxtLink></li>
+    </ul>
+  </nav>
+</template>
+<script>
+export default {
+  name: 'Navigation',
+}
+</script>
+
+<style>
+.button--green {
+  display: inline-block;
+  border-radius: 4px;
+  border: 1px solid #3b8070;
+  color: #3b8070;
+  text-decoration: none;
+  padding: 10px 30px;
+}
+
+.button--green:hover {
+  color: #fff;
+  background-color: #3b8070;
+}
+
+.button--grey {
+  display: inline-block;
+  border-radius: 4px;
+  border: 1px solid #35495e;
+  color: #35495e;
+  text-decoration: none;
+  padding: 10px 30px;
+  margin-left: 15px;
+}
+
+.button--grey:hover {
+  color: #fff;
+  background-color: #35495e;
+}
+
+.navigation {
+  position: absolute;
+  padding-top: 15px;
+}
+
+.navigation ul {
+  list-style: none;
+  display: flex;
+  flex-direction: row;
+}
+</style>
+
+```
+
+## Page
+
+The next thing is to add a new page.
+We create a `pages` directory where we add the page.
+
+In the new page component, we add our previously created `Wrapper` component.
+
+```vue [@Core/pages/index/index.vue]
+<template>
+  <Wrapper>
+    <h1 class="title">VueMS-demo</h1>
+  </Wrapper>
+</template>
+
+<script>
+import Wrapper from '@Core/components/Wrapper'
+
+export default {
+  name: 'IndexPage',
+  components: {
+    Wrapper,
+  },
+}
+</script>
+<style>
+.title {
+  font-family: 'Quicksand', 'Source Sans Pro', -apple-system, BlinkMacSystemFont,
+    'Segoe UI', Roboto, 'Helvetica Neue', Arial, sans-serif;
+  display: block;
+  font-weight: 300;
+  font-size: 100px;
+  color: #35495e;
+  letter-spacing: 1px;
+}
+</style>
+
+```
+
+## Router
+
+For the page we created in the previous step to work, we need to add routing to it.
```

Comment:
Same here, again repeated same things. Try to point what is the most important in these section.

Reply:
The cookbook is meant to show the step-by-step creation process, so I refer to previously created items to create hooks.